src/components/Footer/Footer.jsx

```diff
@@ -0,0 +1,43 @@
+import React from 'react';
+import Icon from '../Icon/Icon';
+import './Footer.css'
+
+const Footer = () => {
+  const descriptions =['How Triphouse works','Careers','Privacy','Terms'];
+  const homesTypes =['Guest houses','Hotels','Apartments','Villas','Holiday homes','Hostels'];
+
+  const descriptionList = descriptions.map((item, i)=>{
+    return (<li className="footer-item" key={i}>{item}</li>)});
+
+  const homesTypesList = homesTypes.map((item, i)=>{
```

Comment:
descriptionList and homesTypesList  look the same. can you use a component for them?

Reply:
Added List component.